internal/notification/types/user_email.go

```diff
@@ -27,7 +28,9 @@ func generateEmail(
 	emailChannels, config, err := channels.Email(ctx)
 	logging.OnError(err).Error("could not create email channel")
 	if emailChannels == nil || emailChannels.Len() == 0 {
-		return zerrors.ThrowPreconditionFailed(nil, "PHONE-w8nfow", "Errors.Notification.Channels.NotPresent")
+		return zchannels.NewCancelError(
+			zerrors.ThrowPreconditionFailed(nil, "MAIL-w8nfow", "Errors.Notification.Channels.NotPresent"),
```

Comment:
i think twilio is email

```suggestion
			zerrors.ThrowPreconditionFailed(nil, "TWILI-w8nfow", "Errors.Notification.Channels.NotPresent"),
```

Reply:
same for other errors returned

Reply:
I would either use `TWILI` or `PHONE`

Reply:
this is the user_email.go... someone (probably me) already had the great idea of using PHONE here 😉